### PR TITLE
Preview: MR components

### PR DESCRIPTION
**Pull request by atneik**
#### Description of changes

Adding a new package `@uifabric/mr`

#### Focus areas to test

- Dependencies
- Architecture
- Best Practices

### CODE REVIEW

**Comment by JasonGore** [comment]
What are MR components? I wonder if this short package name is descriptive enough.

**Comment by atneik** [comment]
> What are MR components? I wonder if this short package name is descriptive enough.

We could expand and call it `@uifabric/mixed-reality`

File: packages/mr/README [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# [Office UI Fabric React Mr](http://dev.microsoft.com/fabric)
+
+##### Mr components for Office UI Fabric React
```
Comment (Jahnp) [inline packages/mr/README]: As @JasonGore points out in his comment, this should be clarified to something like "Mixed-reality components for..."

**Comment by Jahnp** [comment]
Very cool @atneik ! I'll take a closer look at this soon.

**Comment by atneik** [comment]
Going back to drawing board for certain changes. Will be back

**Pull request closed without merging**